README.md

```diff
@@ -152,6 +162,64 @@ And to run only the client tests, run the test:client task:
 $ grunt test:client
 ```
 
+
+## Running your application with Gulp
+
+After the install process, you can easily run your project with:
+
+```bash
+$ gulp
+```
+or
+
+```bash
+$ gulp default
+```
+
+The server is now running on http://localhost:3000 if you are using the default settings. 
+
+### Errors at Runtime
```

Comment:
I don't think it's a good idea to have a troubleshooting section in the README.
We should have a troubleshooting section in the documentation and point to it from the README